inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
++
+* Tell us what you would like to have
+** INCEpTION is still in developement, so you are welcome to give us
+feedback and tell us your wishes and requirements. For any questions,
+you are welcome to join our Google group „inception users“:
+https://groups.google.com/forum/#!forum/inception-users
++
+* Know our documentation for further reading
+** You will find lots of information in our Guides, called documentation
+(on our website, among the tabs on top: → _Documentation:_
+https://inception-project.github.io/documentation/[https://inception-]https://inception-project.github.io/documentation/[project.github.io/documentation/]
+. There, see the upper section.). If they do not answer your question,
+don‘t hesitate to contact our team: ____ We are glad to help you.
+** Whenever you find a blue questionmark-sign in INCEpTION, you may
+click on it to find further explanation in the main documentation.
+** For more details, see the installation guide
+** For more installation, see the admin guide. You find it on our
+website, among the tabs on top: → _Documentation_,
+https://inception-project.github.io/releases/0.10.0/docs/admin-guide.html[h]https://inception-project.github.io/releases/0.10.0/docs/admin-guide.html[ttps://inception-project.github.io/releases/0.10.0/docs/admin-guide.html]
 
-Alternatively, you can start it on the command line using
+Installing and starting INCEpTION
 
-[subs="+attributes"]
-----
-$ java -jar inception-app-standalone-{revnumber}.jar
-----
+* Installing Java
+** In order to make INCEpTION run, you will need to install Java (in
+case you do not have it yet). Here
+https://java.com/en/download/help/download_options.xml#linux you find
+instructions on how to install Java on Windows, Mac, Linux and Solaris.
 
-The splash screen and dialog will not appear in this case and you have to manually point your browser at `http://localhost:8080`.
+* Download and start INCEpTION
+** Now, download the .jar-file from our website
+https://inception-project.github.io/ . You will find the
+„Download“-Button top right. Wait a moment until it is fully downloaded.
+That is, until the name of the downlauded folder ends on „.jar“, not on
+„.jar.part“ anymore.
++
+_Note_: We recommend working on the latest version always. So check the
+website once in a while and download the new .jar-file when a later
+version is available.
+** Now, simply ***double-click ***on the downloaded .jar-file. After a
+moment, a splash screen will display while the application is
+initializing. Once the initialization is complete, a dialog appears.
+Here, you can open the application in your default browser or shut it
+down again.
+** If you are used to the command line, you may enter this command there
+instead:
++
+$ java -jar inception-app-standalone-0.10.0.jar
++
+In this case, no splash screen will start. Just enter
+http://localhost:8080/[_http://localhost:8080_] in you browser.
+** Use „admin“ as username and password to login, since a default user
```

Comment:
remove `since`

Reply:
The sentence will be broken then. What's your purpose on deleting "since"?

Reply:
since implies causation to me which is not here. I would then better swap the sentence parts:
When you start inception for the first time, a default user admin with password admin is created. Use this user to log into inception.

Reply:
done